api/build.gradle

```diff
@@ -6,7 +6,8 @@ dependencies {
 
     compileOnly libraries.auto_value
 
-    signature "org.codehaus.mojo.signature:java16:+@signature"
+    signature "org.codehaus.mojo.signature:java17:+@signature"
+    signature "net.sf.androidscents.signature:android-api-level-14:+@signature"
```

Comment:
gRPC uses specific version numbers `1.0` and `4.0_r4`, while you use `+` here. Any reason to be different?

Reply:
fixed.